src/dlgAboutDialog.cpp

```diff
@@ -94,4 +104,314 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
     }
 
     mudletTitleLabel->setPixmap(QPixmap::fromImage(splashImage));
+
+    // Have moved the texts in from the dialog definitions - as it makes it
+    // easier to do the translations required for I18n work (Qt Linguist is
+    // "borked" for HTML content particularly in dialogues ):
+
+    slot_guiLanguageChange();
+    connect(mudlet::self(), SIGNAL(signal_translatorChangeCompleted(const QString&, const QString&)), this, SLOT(slot_guiLanguageChange()));
+}
+
+void dlgAboutDialog::slot_guiLanguageChange()
+{
+    retranslateUi(this);
+    // PLACEMARKER: Redefine GUI Texts
+
+    // Define a uniform header for all tabs:
+    QString htmlHead(QStringLiteral("<head><style type=\"text/css\">"
+                                    "h1 { font-family: \"DejaVu Serif\"; text-align: center; }"
+                                    "h2 { font-family: \"DejaVu Serif\"; text-align: center; }"
+                                    "h3 { font-family: \"DejaVu Serif\"; text-align: center; white-space: pre-wrap; }"
+                                    "h4 { font-family: \"DejaVu Serif\" }"
+                                    "p { font-family: \"DejaVu Serif\" }"
+                                    "tt { font-family: \"Monospace\"; white-space: pre-wrap;}"
+                                    "</style></head>"));
+
+    // TAB 1 - "About Mudlet"
+    // I have found the hard way that the lupdate utility to extract translatable
+    // strings from the code does not like C++11 syle raw string literals, see:
+    // https://bugreports.qt.io/browse/QTBUG-42736
+    QString aboutMudletHeader(tr("<tr><td><span style=\"color:#bc8942;\"><b>Homepage</b></span></td><td><a href=\"http://www.mudlet.org/\">www.mudlet.org</a></td></tr>\n"
```

Comment:
How come this was moved here? This significantly obstructs the ability to edit it.

Reply:
Have you seen the HTML that this would get converted to by the QtDesigner plugin 😱 - and a translator - whether using QtLinguist __or an external system__ would get __that__ to work on, it does compromise the ease of editing but it reduces the pain at translation time...